### PR TITLE
Apply new format to snippets and template

### PR DESCRIPTION
**Pull request by Arpit045-boop**

### CODE REVIEW

File: snippets/prime_number.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: prime_number
+tags: prime_number,intermediate
```
Comment (Trinityyi) [inline snippets/prime_number.md:3]: Tags should be `math,beginner`

File: snippets/prime_number.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: prime_number
+tags: prime_number,intermediate
+---
+
+Explain briefly what the snippet does.
```
Comment (Trinityyi) [inline snippets/prime_number.md:6]: Missing description

File: snippets/prime_number.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+title: prime_number
+tags: prime_number,intermediate
+---
+
+Explain briefly what the snippet does.
+
+- Checks numbers from 2 to (given number - 1), If number is divisible by given numbers.
```
Comment (Trinityyi) [inline snippets/prime_number.md:8]: The explanation could use some more work. I suppose the algorithm is similar to the [JS implementation](https://www.30secondsofcode.org/js/s/is-prime), so you could write something similar?

File: snippets/prime_number.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+Explain briefly what the snippet does.
+
+- Checks numbers from 2 to (given number - 1), If number is divisible by given numbers.
+- Or, remainder is 0 than its a not a prime number.
+- If number is prime than return 1 and otherwise return 0. 
+
+```py
+def prime_number(number):
```
Comment (Trinityyi) [inline snippets/prime_number.md:13]: The complexity of this is sub-optimal. Also, I'm pretty sure the `else` part is incorrent. There's a better solution like the second to last snippet [here](https://stackoverflow.com/a/18833870/9298688).

File: snippets/prime_number.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  # code
+  for other_numbers in range(2,number):
+    if(number % other_numbers == 0):
+        return 0
+    else:
+        return 1
+```
+
+```py
+prime_number(21) # result
```
Comment (Trinityyi) [inline snippets/prime_number.md:23]: The comment should show the result.

File: snippets/prime_number.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+Explain briefly what the snippet does.
+
+- Checks numbers from 2 to (given number - 1), If number is divisible by given numbers.
+- Or, remainder is 0 than its a not a prime number.
+- If number is prime than return 1 and otherwise return 0. 
+
+```py
+def prime_number(number):
+  # code
```
Comment (Trinityyi) [inline snippets/prime_number.md:14]: ✂️ this comment please

File: snippets/prime_number.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+---
+title: prime_number
```
Comment (Trinityyi) [inline snippets/prime_number.md:2]: Title should be `is_prime` with the function name matching that.

**Review by Trinityyi — CHANGES_REQUESTED** [review]
Thanks for conributing. Please fix the mentioned issues and we can take another look afterwards.

File: snippets/is_prime.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
+---
+title: is_prime
+tags:  math,beginner
+---
+
```
Comment (Chalarangelo) [inline snippets/is_prime.md:5]: Please add a description.

File: snippets/is_prime.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+- Check numbers from 2 to the given number, If given number is divisible by numbers.
+- Or, remainder is 0 than its a not a prime number.
+- If number is prime than print "Prime Number" and otherwise print "Not a prime number". 
+
+```py
+def is_prime(number):
+  for other_numbers in range(2,number):
+    if(number % other_numbers == 0):
+        print("not a prime number")
```
Comment (Chalarangelo) [inline snippets/is_prime.md:14]: Incorrect indentation.

Comment (Chalarangelo) [inline snippets/is_prime.md:14]: The function must return a boolean value, not print the result.

File: snippets/is_prime.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+        break
+  else:
```
Comment (Chalarangelo) [inline snippets/is_prime.md:16]: The `else` is wrong. This should return after the loop is finished.

File: snippets/is_prime.md [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+    print("Prime number")
+```
+
+```py
+is_prime(21)
```
Comment (Chalarangelo) [inline snippets/is_prime.md:21]: This should be followed by a comment with the result.

**Review by Chalarangelo — CHANGES_REQUESTED** [review]

**Review by Chalarangelo — CHANGES_REQUESTED** [review]
> The complexity of this is sub-optimal. Also, I'm pretty sure the `else` part is incorrent. There's a better solution like the second to last snippet [here](https://stackoverflow.com/a/18833870/9298688).

The suggested implementation here looks significantly more performant, could you use that instead?

**Review by Chalarangelo — COMMENTED** [review]
Description, explanation and example are still wrong. Closing due to lack of following the guidelines and having to constantly review this over and over line by line.

**Pull request closed without merging**